Metadata.md

```diff
@@ -112,7 +112,7 @@ name: esp32
 An example:
 
 ```yaml
-name: BSD-3-Clause
+name: BSD-3
```

Comment:
what is the motivation of this? if it's too long to display, the view, or `README.md.erb`, should do this. there are several BSD-3-* licenses.

Reply:
Yes, it seems a bit long to me. I replaced `BSD-3-Clause` with `BSD-3` in the whole project and fixed "License" column length in the `README.md.erb`.